rasa/engine/graph.py

```diff
@@ -464,9 +464,15 @@ def __call__(
             # handling of exceptions.
             raise
         except Exception as e:
-            raise GraphComponentException(
-                f"Error running graph component for node {self._node_name}."
-            ) from e
+            if not isinstance(e, RasaException):
+                raise GraphComponentException(
+                    f"Error running graph component for node {self._node_name}."
+                ) from e
+            else:
+                logger.error(
+                    f"Error running graph component for node" f" {self._node_name}."
```

Comment:
Is this meant to be two `f""`s?

Reply:
nope, this was `make formatter` deciding to leave some work for me